Mock_Jetson.py

```diff
@@ -0,0 +1,119 @@
+from Client import *
+import time
+from SensorState import *
+import RandomObjects
+from RandomObjects import RandomObjects
+import GenerateSensorData
+import grid
+from tkinter import *
+import math
+from Consts import *
+from GenerateSensorData import GenerateSensorData
+from EndpointInput import *
+
+class Mock_Jetson:
+    def __init__(self):
+        """
+        """
+        self.grid = grid.Grid(tile_num_height, tile_num_width, tile_size)
+        self.client = Client()
+        self.pos = (0, 0)
+        self.sensor_state = SensorState()
+
+        #Visualization
+        self.master: Tk = Tk()
+        self.canvas: Canvas = None
+        self.heading = 0
+        self.prev_draw_c1c0_ids = [None, None]
+        self.tile_dict: Dict[grid.Tile, int] = None
+
+        # Generates random enviroment on the grid
+        generator = RandomObjects(self.grid)
+        generator.create_env(22, 0, 0, 22, 9)
+        self.sensor_generator = GenerateSensorData(self.grid)
+
+        self.create_widgets()
+        # starting location for middle
+        self.curr_tile = self.grid.grid[int(tile_num_width/2)][int(tile_num_height/2)]
+
+        self.client.send_data({'end_point': userInput()})
+        self.main_loop()
+        self.master.mainloop()
+
+    def main_loop(self):
+        """
+        """
+        # TODO: update the curr tile from read
+        new_coor = self.client.listen()
+        self.curr_tile = self.grid.grid[new_coor[0]][new_coor[1]]
+        self.drawC1C0()
+        self.sensor_state.lidar = self.sensor_generator.generateLidar(2, self.curr_tile.row, self.curr_tile.col)
```

Comment:
We changed this to 2 degrees... can we change it back to 1 degrees and will it still work?

Reply:
Resolving